### PR TITLE
Add user blocking tests.

### PR DESCRIPTION
**Pull request by hoolioh**
## Description
Add user blocking tests by making a requests with a 'user' query parameter to an endpoint '/users'. Rules for blocking the users are hardcoded in blocking_rules.json so the tests are run under APPSEC_BLOCKING scenario 

## Check list

Your PR is not ready to be reviewed? Please save it as a draft :pray:

- [x] Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
- [x] CI is passing
- [x] There is at least one approval from code owners

Yes to all? Feel free to merge it whenever you want :heart:

### CODE REVIEW

File: utils/build/docker/nodejs/express4/app.js [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 const tracer = require("dd-trace").init({
   debug: true,
+  appsec: {
```
Comment (uurien) [inline utils/build/docker/nodejs/express4/app.js:5]: why add this?
We already had some appsec related test that were working, may already be activated by environment var?

Comment (hoolioh) [reply utils/build/docker/nodejs/express4/app.js]: left it by mistake

File: utils/build/docker/nodejs/express4/app.js [hunk 2]
```diff
@@ -87,6 +87,22 @@ app.get("/make_distant_call", (req, res) => {
...
+  } else {
+    user.id = 'anonymous'
+  }
+
+  const ret = tracer.appsec.isUserBlocked(user)
+  if (ret) {
+    tracer.appsec.blockRequest(req, res)
+  } else {
+    res.send(`Hello ${user.id}`)
+  }
```
Comment (uurien) [inline utils/build/docker/nodejs/express4/app.js:103]: Let's try to not use var names like `ret`
```suggestion
  const shouldBlock = tracer.appsec.isUserBlocked(user)
  if (shouldBlock) {
    tracer.appsec.blockRequest(req, res)
  } else {
    res.send(`Hello ${user.id}`)
  }
```

File: docs/weblog/README.md [hunk 3]
```diff
@@ -196,3 +196,12 @@ This endpoint calls the appsec event tracking SDK function used for custom event
...
 - Event name: `system_tests_event`
 - Metadata: `{metadata0: value0, metadata1: value1}`
+
+## GET /users
+
+This endpoint calls the appsec blocking SDK functions used for blocking users. If the expected parameter matches one of
+the possible values the WAF will return the proper action.
+
+Expected query parameters:
+- `user`: user name.
```
Comment (simon-id) [inline docs/weblog/README.md:206]: user id not name right  ?

**Review by cbeauchesne — APPROVED** [review]

File: docs/weblog/README.md (hunk 3 above)
Comment (hoolioh) [reply docs/weblog/README.md]: You're right.

**Review by simon-id — APPROVED** [review]

**Review by smola — APPROVED** [review]

**Pull request merged**